lib/auth/oauth_faraday_formatter.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+class Auth::OauthFaradayFormatter < Faraday::Logging::Formatter
+  def request(env)
+    warn <<~LOG
+      OAuth Debugging: request #{env.method.upcase} #{env.url}
+
+      Headers:
+      #{env.request_headers}
+
+      Body:
+      #{env[:body]}
+    LOG
+  end
+
+  def response(env)
+    warn <<~LOG
+      OAuth Debugging: response status #{env.status}
+
+      From #{env.method.upcase} #{env.url}
+
+      Headers:
+      #{env.request_headers}
```

Comment:
I feel like this should be 'response_headers'? (so... I guess this is wrong in all the other plugins... ooops)